Makefile

```diff
@@ -378,130 +584,88 @@ remoteapiv2:
+	$(eval TMPDIR := $(shell mktemp -d podman_tmp_XXXX))
 	$(eval SUBDIR := podman-v$(RELEASE_NUMBER))
 	mkdir -p "$(TMPDIR)/$(SUBDIR)"
-	$(MAKE) install.bin install.man install.cni install.systemd "DESTDIR=$(TMPDIR)/$(SUBDIR)" "PREFIX=/usr"
-	# release.txt location and content depended upon by automated tooling
-	cp release.txt "$(TMPDIR)/"
-	tar -czvf $@ --xattrs -C "$(TMPDIR)" "./release.txt" "./$(SUBDIR)"
+	$(MAKE) install.bin install.man install.cni \
+		install.systemd "DESTDIR=$(TMPDIR)/$(SUBDIR)" "PREFIX=/usr"
+	tar -czvf $@ --xattrs -C "$(TMPDIR)" "./$(SUBDIR)"
 	-rm -rf "$(TMPDIR)"
 
-# Must call make in-line: Dependency-spec. w/ wild-card.
-podman-remote-release-%.zip:
-	$(MAKE) podman-remote-$* install-podman-remote-$*-docs release.txt \
-		RELEASE_BASENAME=$(shell hack/get_release_info.sh REMOTENAME) \
-		RELEASE_DIST=$* RELEASE_DIST_VER="-"
-	$(eval TMPDIR := $(shell mktemp -d -p '' $podman_remote_XXXX))
-	$(eval SUBDIR := podman-$(RELEASE_VERSION))
-	$(eval BINSFX := $(shell test "$*" != "windows" || echo ".exe"))
+podman-remote-release-%.zip: podman-remote-% install-podman-remote-%-docs  ## Build podman-remote for GOOS=%, docs., and installation zip.
+	$(eval TMPDIR := $(shell mktemp -d podman_tmp_XXXX))
+	$(eval SUBDIR := podman-$(RELEASE_NUMBER))
 	mkdir -p "$(TMPDIR)/$(SUBDIR)"
-	# release.txt location and content depended upon by automated tooling
-	cp release.txt "$(TMPDIR)/"
-	cp ./bin/podman-remote-$*$(BINSFX) "$(TMPDIR)/$(SUBDIR)/podman$(BINSFX)"
+	$(MAKE) \
+		GOOS=$* \
+		DESTDIR=$(TMPDIR)/ \
+		BINDIR=$(SUBDIR) \
+		SELINUXOPT="" \
+		install.remote-nobuild
 	cp -r ./docs/build/remote/$* "$(TMPDIR)/$(SUBDIR)/docs/"
 	cp ./contrib/remote/containers.conf "$(TMPDIR)/$(SUBDIR)/"
-	cd "$(TMPDIR)/$(SUBDIR)" && \
-		zip --recurse-paths "$(CURDIR)/$@" "./release.txt" "./"
+	cd "$(TMPDIR)" && \
+		zip --recurse-paths "$(CURDIR)/$@" "./"
 	-rm -rf "$(TMPDIR)"
 
-.PHONY: podman-release
-podman-release:
-	rm -f release.txt
-	$(MAKE) podman-release.tar.gz
-
-.PHONY: podman-remote-%-release
-podman-remote-%-release:
-	rm -f release.txt
-	$(MAKE) podman-remote-release-$*.zip
+.PHONY: podman.msi
+podman.msi: podman-v$(RELEASE_NUMBER).msi  ## Build podman-remote, package for installation on Windows
+podman-v$(RELEASE_NUMBER).msi: podman-remote-windows install-podman-remote-windows-docs
+	$(eval DOCFILE := docs/build/remote/windows)
+	find $(DOCFILE) -print | \
+		wixl-heat --var var.ManSourceDir --component-group ManFiles \
+		--directory-ref INSTALLDIR --prefix $(DOCFILE)/ > \
+			$(DOCFILE)/pages.wsx
+	wixl -D VERSION=$(RELEASE_VERSION) -D ManSourceDir=$(DOCFILE) \
+		-o $@ contrib/msi/podman.wxs $(DOCFILE)/pages.wsx
 
-.PHONY: generate-bindings
-generate-bindings:
-ifneq ($(shell uname -s), Darwin)
-	GO111MODULE=off $(GO) generate ./pkg/bindings/... ;
-endif
+.PHONY: package
+package:  ## Build rpm packages
+	## TODO(ssbarnea): make version number predictable, it should not change
+	## on each execution, producing duplicates.
```

Comment:
Should `scrape_version()` be exposed to take care of this TODO?

Reply:
Oh, good catch.  I have no idea why specifically that TODO was added, but from the text it seems we could remove it.

Reply:
This came in by https://github.com/containers/podman/pull/4601 

I looked through `contrib/build_rpm.sh` but don't see any references or direct use of `hack/get_release_info.sh` or scraping one of the `podman-*-release` target filenames.

@ssbarnea is this `TODO` in the Makefile `package:` target relevant to any of the release targets present in the Makefile or just something internal for the `package:` target?

Reply:
(Let's not hold up this PR for the question...it's not worth re-running all the tests just to remove a `TODO` - if necessary).